tests/test_invoice.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from pyzbar.pyzbar import decode
+
+from apps.payments.invoice import format_inline_epc_qr
+from models.payment import BankPayment
+
+from tests._utils import render_svg
+
+
+@pytest.mark.skip
+def test_format_inline_epc_qr():
+    payment = BankPayment(currency="EUR", amount=10)
+
+    qr_inline = format_inline_epc_qr(payment)
+    qr_image = render_svg(qr_inline)
+
+    expected = [
+        "BCD",
+        "002",
+        "1",
+        "SCT",
+        "",
+        "Electromagnetic Field Ltd",
+        "GB21BARC20716472954433",
```

Comment:
These are our live bank details, which we shouldn't have in git. (It's a minor thing but people do occasionally harvest them for direct debit fraud.)

The test Euro IBAN is this one:

https://github.com/emfcamp/Website/blob/e8276d3d7f32fcaf9dd763cbe4d91a0f22a0fe0e/apps/base/tasks_banking.py#L43